x-pack/test/encrypted_saved_objects_api_integration/tests/encrypted_saved_objects_api.ts

```diff
@@ -683,7 +734,50 @@ export default function ({ getService }: FtrProviderContext) {
             .expect(200);
           const { id: objectId } = savedObject;
 
-          const getApiUrl = getGetApiUrl({ objectId, spaceId });
+          const getApiUrl = getGetApiUrl({
+            type: 'saved-object-with-migration',
+            objectId,
+            spaceId,
+          });
+          const { body: decryptedResponse } = await supertest.get(getApiUrl).expect(200);
+          expect(decryptedResponse.attributes).to.eql(expectedDecryptedAttributes);
+        });
+      });
+
+      // In these test cases, we simulate a scenario where some existing model version objects need to migrated. This happens because they
```

Comment:
__typo:__
```suggestion
      // In these test cases, we simulate a scenario where some existing model version objects need to be migrated. This happens because they
```